components/integrations/confirm_integration/confirm_integration.jsx

```diff
@@ -51,152 +51,148 @@ export default class ConfirmIntegration extends React.Component {
         let tokenText = null;
 
-        if (this.state.type === Constants.Integrations.COMMAND) {
-            const command = this.props.commands[this.state.id];
-            if (command) {
-                headerText = (
-                    <FormattedMessage
-                        id={'installed_commands.header'}
-                        defaultMessage='Slash Commands'
+        const command = this.props.commands[this.state.id];
+        const incomingHook = this.props.incomingHooks[this.state.id];
+        const outgoingHook = this.props.outgoingHooks[this.state.id];
+        const oauthApp = this.props.oauthApps[this.state.id];
+
+        if (this.state.type === Constants.Integrations.COMMAND && command) {
+            headerText = (
+                <FormattedMessage
+                    id={'installed_commands.header'}
+                    defaultMessage='Slash Commands'
+                />
+            );
+            helpText = (
+                <p>
+                    <FormattedMarkdownMessage
+                        id='add_command.doneHelp'
+                        defaultMessage='Your slash command has been set up. The following token will be sent in the outgoing payload. Please use it to verify the request came from your Mattermost team (see [documentation](!https://docs.mattermost.com/developer/slash-commands.html) for further details).'
                     />
-                );
-                helpText = (
-                    <p>
-                        <FormattedMarkdownMessage
-                            id='add_command.doneHelp'
-                            defaultMessage='Your slash command has been set up. The following token will be sent in the outgoing payload. Please use it to verify the request came from your Mattermost team (see [documentation](!https://docs.mattermost.com/developer/slash-commands.html) for further details).'
-                        />
-                    </p>
-                );
-                tokenText = (
-                    <p className='word-break--all'>
-                        <FormattedHTMLMessage
-                            id='add_command.token'
-                            defaultMessage='<b>Token</b>: {token}'
-                            values={{
-                                token: command.token,
-                            }}
-                        />
-                    </p>
-                );
-            }
-        } else if (this.state.type === Constants.Integrations.INCOMING_WEBHOOK) {
-            const incomingHook = this.props.incomingHooks[this.state.id];
-            if (incomingHook) {
-                headerText = (
-                    <FormattedMessage
-                        id={'installed_incoming_webhooks.header'}
-                        defaultMessage='Incoming Webhooks'
+                </p>
+            );
+            tokenText = (
+                <p className='word-break--all'>
+                    <FormattedHTMLMessage
```

Comment:
Non-blocking and is not be related but would be nice in this component to convert all instances of `FormattedHTMLMessage` into `FormattedMarkdownMessage`.  Like in this case, it should have `defaultMessage='**Token**: {token}'`.  Should update as well the value at https://github.com/mattermost/mattermost-webapp/blob/2dc3d34b2007cc3f66d885f97dd464e705f47a29/i18n/en.json#L61.
Let me know that sounds good to you.  Thanks!